src/main/java/bh/bot/app/ChangeCharacterApp.java

```diff
@@ -0,0 +1,142 @@
+package bh.bot.app;
+
+import static bh.bot.common.Log.info;
+import static bh.bot.common.Log.debug;
+import static bh.bot.common.utils.InteractionUtil.Mouse.moveCursor;
+import static bh.bot.common.utils.ThreadUtil.sleep;
+
+import java.awt.Point;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import bh.bot.Main;
+import bh.bot.common.Configuration;
+import bh.bot.common.Telegram;
+import bh.bot.common.exceptions.NotSupportedException;
+import bh.bot.common.types.Offset;
+import bh.bot.common.types.annotations.AppMeta;
+import bh.bot.common.types.annotations.RequireSingleInstance;
+import bh.bot.common.types.images.BwMatrixMeta;
+import bh.bot.common.utils.ColorizeUtil;
+import bh.bot.common.utils.InteractionUtil;
+import bh.bot.common.utils.ThreadUtil;
+
+@AppMeta(code = "character", name = "Change Character", displayOrder = 1, argType = "number", argAsk = "What character slot do you want to pick?", argDefault = "1", argRequired = true)
+@RequireSingleInstance
+public class ChangeCharacterApp extends AbstractApplication {
+
+    @Override
+    protected void internalRun(String[] args) {
+        int arg;
+        try {
+            arg = Integer.parseInt(args[0]);
+        } catch (ArrayIndexOutOfBoundsException | NumberFormatException ex) {
+            info(getHelp());
+            arg = readInputLoopCount("What character slot do you want to pick?");
+        }
+
+        final int characterSlot = arg;
+        info("Character in slot %3d ", characterSlot);
+        AtomicBoolean masterSwitch = new AtomicBoolean(false);
+        ThreadUtil.waitDone(
+                () -> doLoopClickImage(characterSlot, masterSwitch),
+                () -> internalDoSmallTasks( //
+                        masterSwitch, //
+                        SmallTasks //
+                                .builder() //
+                                .clickTalk() //
```

Comment:
are those option needed? ClickTalk,ClickDisconnect,ReactiveAuto,....?

Reply:
No clue! Didn't want to break anything :P I just started using your bot on Friday

Reply:
But you adopt the code base so fast, really impressive, ty